### PR TITLE
Modify getEvents()

### PR DESCRIPTION
**Pull request by fang-helen**
Modified getEvents to choose which footer item to display -- a reason for recommendation, a link to edit/unsave, or the attendee count.
This now takes in 3 parameters instead of one, including an index (if there are multiple event lists on a single page, such as my-events.html) as well as an option specifying which of these footer items to display.
Depending on the parameters this also decides whether or not to only display a distance away, or the address and the time.

### CODE REVIEW

**Review by nmsulliv — APPROVED** [review]
Looks good!

File: src/main/webapp/script.js [hunk 1]
```diff
@@ -176,7 +193,11 @@ async function getEvents(url) {
     eventItemDetailsElement.appendChild(eventItemDateElement);
     const eventItemDistanceElement = document.createElement('div');
     eventItemDistanceElement.className = 'event-item-distance';
-    eventItemDistanceElement.innerText = event.distance;
+    if(option == 1 || option == 2) {
```
Comment (0tpyrk) [inline src/main/webapp/script.js]: Space between if and ( ?

File: src/main/webapp/script.js [hunk 2]
```diff
@@ -188,16 +209,44 @@ async function getEvents(url) {
...
+    // decide which footer item to use
     const attendeeCountContainerElement = document.createElement('div');
-    attendeeCountContainerElement.className = 'attendee-count-container';
-    eventItemFooterElement.appendChild(attendeeCountContainerElement);
-    const attendeeCountElement = document.createElement('span');
-    attendeeCountElement.className = 'attendee-count ' + event.tags[0] + '-text';
-    attendeeCountElement.innerText = event.attendeeCount;
-    attendeeCountContainerElement.appendChild(attendeeCountElement);
-    attendeeCountContainerElement.appendChild(
+    if(option == 0) {
```
Comment (0tpyrk) [inline src/main/webapp/script.js]: Same as above

**Review by 0tpyrk — APPROVED** [review]
LGTM, great comments!

**Pull request merged**